appinventor/components-ios/src/Form.swift

```diff
@@ -25,6 +27,7 @@ let kMinimumToastWait = 10.0
   fileprivate var _aboutScreen: String?
   fileprivate var _appName: String?
   fileprivate var _accentColor: Int32 = Int32(bitPattern: 0xFFFF4081)
+  fileprivate var _defaultFileScope: fileScope.App //declare vs define
```

Comment:
Shouldn't this be `_defaultFileScope = FileScope.App`?

Reply:
Also, you need to implement the getter/setter pair for exposing this to other components and the designer.